### DISCUSSION TITLE
taskbar apperance

### DISCUSSION THREAD

**Discussion by vandelin586** [valinet/ExplorerPatcher#2725] [General]
My taskbar is solid white , I cannot even see the start button or clock
![clock](https://github.com/valinet/ExplorerPatcher/assets/19674587/1a50c22d-331c-4907-9ac5-663ea5b8ae9f)

**Comment by pyrates999** [comment]
Please post windows version and build.  You can find this by running winver.
Please post explorer patcher version.

Please include what settings you changed in explorer patcher properties application.

Please include any other software you have installed that modifies the UI of windows.

**Reply by vandelin586** [reply]
windows 11 23H2 22635.3066 
Explorer patch 22621.2861.62.2
I changed nothing in explorer patcher properties to cause this disruption

I did however install this update that i believe is what started this disruption:
Update      KB5034209 1/20/2024 0:00:00
[The update KB5034209 is officially known as Windows 11 Insider Preview Build 22635.3066]

**Comment by vandelin586** [comment]
weird stuff happens with my taskbar lol 
![clock](https://github.com/valinet/ExplorerPatcher/assets/19674587/7444575a-e4d6-4552-828c-e46a79a986c2)

**Comment by pyrates999** [comment]
You're running an insider preview build: https://blogs.windows.com/windows-insider/2024/01/19/announcing-windows-11-insider-preview-build-22635-3066-beta-channel/

Expect stuff to break.

In the settings app, untick "Get the latest updates as soon as available" so that preview updates are not automatically installed.

Uninstall it and don't install updates that are optional.

**Reply by vandelin586** [reply]
I mean it may be optional as in do you want to install it or not press go to continue... it is NOT an optional update under the Microsoft terminology / classification. But i understand as my other computer is prompting do i want to install "windows 11 23H2 22635.3066" Ill hold it off as long as i can since this computer is where it really matters, other computer is test. Ill contact @MikeSW17 on their comment to see if I can use a version to get by ... thanks for the assistance

**Reply by pyrates999** [reply]
Yes it is optional.  If it doesn't automatically install itself, you don't have to install it.  It means it is also a preview update.

**Reply by vandelin586** [reply]
your correct

**Comment by MikeSW17** [comment]
I **had** similar problem with EP and Insider Beta Windows. Latest Symbols for EP just downloaded for 10.0.22635.3066 and problem seems **Fixed**.

**Reply by vandelin586** [reply]
Can you further explain what you mean? You just installed a certain version of explorer patcher?
